src/components/team-member-form/TeamMemberForm.tsx

```diff
@@ -55,13 +62,13 @@ function TeamMemberForm({
     setCurrentClient('');
   };
 
-  const handleCancel = event => {
+  const handleCancel = (event): void => {
```

Comment:
I'd suggest to add the event type here. @RobbinHabermehl Is this appropriate to do so here?

Reply:
I removed the 'event' argument since there is not really an event argument required for this function